config/optional/config/optional/rest.resource.entity.node_type.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 25aef6ba-752b-4b62-9140-69043c383363
+langcode: en
+status: true
+dependencies:
+  module:
+    - basic_auth
+    - node
+    - serialization
+id: entity.node_type
+plugin_id: 'entity:node_type'
+granularity: resource
+configuration:
+  methods:
+    - GET
+  formats:
+    - json
+  authentication:
+    - basic_auth
```

Comment:
This file is a duplicate and should be deleted.

Reply:
Done! Sorry about that.

Reply:
Sorry one sec... GitHub is being weird

Reply:
Ok I think I got it. Please confirm